### PR TITLE
fix: Back Navigation icon color

### LINKED ISSUE #702: Back button color change in various Activity
**Summary:**

Activity like Setting , Scan QR code etc have white color icon for back arrow.

**Steps to reproduce:**

Open Setting Activity,Bank Details etc

**Expected behavior:**

Back Button Color should be black as in activity due to matching in background color and icon color

**Observed behavior:**

Back icon not visible due color variation.

**Screenshots:**

Screenshot_2020-02-05-14-02-13-030_org mifos mobilewallet mifospay

### PR DESCRIPTION
**Pull request by refactor-droidyy**
Fixes #702 

Please Add Screenshots If any UI changes.

Please Add Summary of what changes you have made.(Optional)

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
@refactor-droidyy  Please add relevant screen shot, proper description for the black icon

**Comment by refactor-droidyy** [comment]
**Previous**

Screenshot_2020-02-06-12-43-44-198_org mifos mobilewallet mifospay
Screenshot_2020-02-06-12-43-50-816_org mifos mobilewallet mifospay
Screenshot_2020-02-06-12-43-58-658_org mifos mobilewallet mifospay
Screenshot_2020-02-06-12-44-06-605_org mifos mobilewallet mifospay

**After**
Screenshot_2020-02-06-14-38-45-254_org mifos mobilewallet mifospay
Screenshot_2020-02-06-14-38-51-301_org mifos mobilewallet mifospay
Screenshot_2020-02-06-14-39-02-011_org mifos mobilewallet mifospay
Screenshot_2020-02-06-14-39-07-757_org mifos mobilewallet mifospay

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/base/BaseActivity.java [hunk 1]
```diff
@@ -145,6 +146,12 @@ public void hideBackButton() {
         }
     }
 
+    @Override
+    public void showWhiteBackButton() {
+        showHomeButton();
+        setToolbarIcon(R.drawable.ic_arrow_back_black_24dp);
```
Comment (naman14) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/base/BaseActivity.java]: the function name is showWhiteBackButton but it sets a black icon?
Anyhow, the function name should be `showBackButton`

Comment (refactor-droidyy) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/base/BaseActivity.java]: @naman14 i am sorry i changed the color instead in ic_arrow_back_black_24dp.
Also also in mobile verification activity the back navigation color is white that's why i have used 2 function showWhiteBackButton for white back navigation icon and showBackButton for showing back color arrow navigation button.

Comment (refactor-droidyy) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/base/BaseActivity.java]: @naman14 should I change the name of the function to the corresponding color for back navigation as showBlackBackButton and for white showWhiteBack Button.

**Comment by refactor-droidyy** [comment]
@naman14 is this pr need more changes .

**Comment by refactor-droidyy** [comment]
@naman14 @shiv07tiwari please review

**Comment by refactor-droidyy** [comment]
@shiv07tiwari does this pr needs more changes ?

**Comment by refactor-droidyy** [comment]
@luckyman20 yes we are using this function in the Mobile Registration Screen.
that is why I created a separate function for showing white back button

oie_flPqArJX459c

File: mifospay/src/main/res/drawable/ic_arrow_back_white_24dp.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="@color/white"
```
Comment (luckyman20) [inline mifospay/src/main/res/drawable/ic_arrow_back_white_24dp.xml]: formatting

**Comment by refactor-droidyy** [comment]
@luckyman20, please review

**Comment by refactor-droidyy** [comment]
@luckyman20 is this PR need more changes?

**Comment by refactor-droidyy** [comment]
@shiv07tiwari created a single function that can deal with different colour change.

**Comment by refactor-droidyy** [comment]
@shiv07tiwari is this PR needs more chanages

**Comment by refactor-droidyy** [comment]
@luckyman20 please review.

**Comment by devansh-299** [comment]
@refactor-droidyy this PR has multiple commits. Please squash your commits

**Pull request closed without merging**